docs/sink-owners.md

```diff
@@ -27,6 +27,7 @@ List of Owners
 | --------------- | ------------------ | -------------------| --------------------------------------------- | -------------- |
 | ElasticSearch   | :heavy_check_mark: | :heavy_check_mark: | @AlmogBaku / @andyxning / @huangyuqi          | :ok:           |
 | GCM             | :heavy_check_mark: | :x:                | @kubernetes/heapster-maintainers              | :ok:           |
+| Stackdriver     | :heavy_check_mark: | :x:                | @kubernetes/heapster-maintainers              | :new: #1538    |
```

Comment:
:new: means it's in development/not-yet-merged this should probably just be :ok: